home/models.py

```diff
@@ -9,11 +10,13 @@
 
 class HomePage(Page):
     body = RichTextField(blank=True)
+    lookingfor = RichTextField(blank=True)
     alphatext = RichTextField(blank=True)
     footer = RichTextField(blank=True)
 
     content_panels = Page.content_panels + [
         FieldPanel('body', classname="full"),
+        FieldPanel('lookingfor', classname="full"),
```

Comment:
Some help text here (and maybe some for 'alphatext' too) might be a good idea, just to make it clear what this part of the homepage is